corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -312,14 +321,24 @@ cloudCare.CaseDetailsView = Backbone.View.extend(cloudCare.caseViewMixin).extend
     },
     
     render: function () {
-        var self = this;
+        var self = this,
+            $panelBody,
+            $panel;
         if (!self._everRendered) {
-            self.el = $('<section />').attr("id", "case-details").addClass("span5");
+            self.el = $('<section />').attr("id", "case-details").addClass("col-sm-5");
             self._everRendered = true;
         }
         $(self.el).html(""); // clear
+
+        $panel = $('<div class="panel panel-default"></div>')
+            .append('<div class="panel-heading">Case Details</div>');
+        $panelBody = $('<div class="panel-default"></div>');
+        $panel.append($panelBody);
+        $(self.el).append($panel);
+
+
         if (self.model) {
-            var table = $("<table />").addClass("table table-striped datatable").appendTo($(self.el));
+            var table = $("<table />").addClass("table table-striped datatable").appendTo($panelBody);
```

Comment:
no `table-hover`? (I rather wish table-hover behavior was part of the default table class.)